provision-osx/defaults/main.yml

```diff
@@ -40,7 +42,28 @@ homebrew_taps:
 - homebrew/core
 - caskroom/cask
 
-xcode_install_user:
-xcode_install_password:
 xcode_versions:
-- 8
\ No newline at end of file
+- 8.3
+
+buildfarm_node_host: "{{ ansible_host }}"
+
+buildfarm_node_port: 22
+
+buildfarm_node_root_dir: ~/digger
+
+buildfarm_credential_id: macOS_buildfarm_cred
+
+buildfarm_credential_description: Shared credential for the macOS nodes in the buildfarm
+
+buildfarm_node_name: "macOS ({{ ansible_host }})"
+
+buildfarm_node_labels:
+- ios
+
+buildfarm_user_id: admin
+
+buildfarm_node_executors: 1
+
+buildfarm_node_mode: NORMAL
+
+buildfarm_node_description: macOS node for the buildfarm
```

Comment:
some comments here to explain those configuration would be helpful. I guess admin users will have to override some of the fields?